docs/nsm_api.md

```diff
@@ -146,7 +146,7 @@ The common pattern for negotiation of all of these matters between a NSM1 and NS
 message RemoteConnectionRequest {
    required string request_id = 1;
    required string network_service_name = 2;
-   required string nse_provider_name = 4;
+   optional string nse_name = 4;
```

Comment:
The message should be structured to keep all "non-optional" fields first and them possibly fields which could be omitted.

Reply:
Good point, will do.

Reply:
Also proto 3 does not have required either, so all `required` can be dropped.